actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -7,7 +7,11 @@
 
 module AbstractController
   class DoubleRenderError < Error
-    DEFAULT_MESSAGE = "Render and/or redirect were called multiple times in this action. Please note that you may only call render OR redirect, and at most once per action. Also note that neither redirect nor render terminate execution of the action, so if you want to exit an action after redirecting, you need to do something like \"redirect_to(...) and return\"."
+    DEFAULT_MESSAGE = "Render and/or redirect were called multiple times in this action. " \
```

Comment:
For breaking up strings, you could also use a heredoc and [squish](https://edgeapi.rubyonrails.org/classes/String.html#method-i-squish).

Reply:
These are cosmetic changes which we don’t accept.

Reply:
Yup. Can you please revert. If you want to fix typos, it is fine. If you want to change the way the code looks, please don't.